common/src/main/java/dev/lavalink/youtube/track/YoutubeAudioTrack.java

```diff
@@ -108,7 +108,12 @@ public void process(LocalAudioTrackExecutor localExecutor) throws Exception {
 
           if ("Not success status code: 403".equals(message) ||
               "Invalid status code for player api response: 400".equals(message) ||
-              (message != null && message.contains("No supported audio streams available"))) {
+                  (message != null && (message.contains("No supported audio streams available") ||
+                          message.contains("Must find action functions from script") ||
```

Comment:
I think we should consider throwing a custom exception for player script extraction errors that can be caught once instead of having all these `contains` checks.